Source/WebCore/platform/graphics/gstreamer/ImageGStreamerSkia.cpp

```diff
@@ -91,7 +91,7 @@ ImageGStreamer::ImageGStreamer(GRefPtr<GstSample>&& sample)
         break;
     }
 
-    auto imageInfo = SkImageInfo::Make(videoFrame->width(), videoFrame->height(), colorType, alphaType);
+    auto imageInfo = SkImageInfo::Make(videoFrame->width(), videoFrame->height(), colorType, alphaType, SkColorSpace::MakeSRGB());
```

Comment:
Here instead of hardcoding sRGB, maybe get the colorimetry info from the GstVideoInfo. There is a `PlatformVideoColorSpace videoColorSpaceFromInfo(const GstVideoInfo&);` function in GStreamerCommon. Then somehow map the PlatformVideoColorSpace to the skia equivalent.

Reply:
I've checked and that conversion does not look trivial, therefore I've created a simple, not-implemented converter. If you don't mind I'll leave that converter for followup PR.